cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/config/InputConfig.java

```diff
@@ -31,6 +32,14 @@ public void setPath(String path) {
         this.path = path;
     }
 
+    @JsonProperty(required = true)
+    public String getRelTo() {
```

Comment:
If such a property is required, could we at least call it something like 'basePath' or 'relativToPath' or something at least somewhat intuitive in the config?

Reply:
Good catch, that should not be required.